examples/project-service-list-simple/.expected/config.yaml

```diff
@@ -0,0 +1 @@
+testType: render
```

Comment:
I think default testype is `render`, so you can omit this file completely.

Reply:
Per docs here https://github.com/GoogleContainerTools/kpt/blob/main/pkg/test/runner/README.md#how-does-it-work it seemed like default was eval.

Reply:
I created https://github.com/GoogleContainerTools/kpt/pull/2492 to fix it.

Reply:
removed testype